app/views/medicaid/intro_citizen/edit.html.erb

```diff
@@ -2,7 +2,9 @@
 
 <div class="form-card">
   <header class="form-card__header">
-    <div class="form-card__title">Are you currently a US Citizen?</div>
+    <div class="form-card__title">
+      Is everyone in your household currently a US citizen?
```

Comment:
do we want to show the other page title if there is just one person in the household? We did that using i18n pluralization for other features